diff --git a/node_modules/marko/dist/translator/index.js b/node_modules/marko/dist/translator/index.js
index 8919f06..707d953 100644
--- a/node_modules/marko/dist/translator/index.js
+++ b/node_modules/marko/dist/translator/index.js
@@ -9439,6 +9439,7 @@ var script_default = {
       tag.remove();
     }
   },
+  html: false,
   parseOptions: {
     text: true,
     preserveWhitespace: true
diff --git a/node_modules/marko/index.d.ts b/node_modules/marko/index.d.ts
index 0aaa015..43f0709 100644
--- a/node_modules/marko/index.d.ts
+++ b/node_modules/marko/index.d.ts
@@ -96,7 +96,7 @@ declare global {
         input: Marko.TemplateInput<Input>,
         reference: Node,
         position?: "afterbegin" | "afterend" | "beforebegin" | "beforeend",
-      ): Marko.MountedTemplate<typeof input, Return>;
+      ): Marko.MountedTemplate<typeof input>;
       /** @marko-overload-end */
     }
 
